tomcat/src/main/java/org/apache/coyote/http11/Http11Processor.java

```diff
@@ -29,19 +40,85 @@ public void process(final Socket connection) {
         try (final var inputStream = connection.getInputStream();
              final var outputStream = connection.getOutputStream()) {
 
-            final var responseBody = "Hello world!";
-
-            final var response = String.join("\r\n",
-                    "HTTP/1.1 200 OK ",
-                    "Content-Type: text/html;charset=utf-8 ",
-                    "Content-Length: " + responseBody.getBytes().length + " ",
-                    "",
-                    responseBody);
+            final var requestData = getRequestData(inputStream);
+            log.info("requestData : {}", requestData);
+            tomcatController.handleRequest(requestData);
+            final var response = getResponse(requestData);
 
             outputStream.write(response.getBytes());
             outputStream.flush();
+
         } catch (IOException | UncheckedServletException e) {
             log.error(e.getMessage(), e);
         }
     }
+
+    private RequestData getRequestData(InputStream inputStream) throws IOException {
+        var inputReader = new BufferedReader(new InputStreamReader(inputStream, StandardCharsets.UTF_8));
```

Comment:
inputReader라는 변수명이 어색해보여요.
Reader 는 외부 -> 자바  입력 방향을 나타내니 input 이라는 이름을 생략해도 좋을 것 같아요.

그래서 reader 또는 bufferedReader와 같은 네이밍이 의미가 명확하고 중복을 피할 수 있어 가독성이 더 좋아질 것 같습니다.

```suggestion
        var bufferedReader = new BufferedReader(new InputStreamReader(inputStream, StandardCharsets.UTF_8));
```

Reply:
그렇네요 !! reader라는 이름 자체가 이미 input을 읽는건데 중복된 표현이였던 것 같습니다.
bufferedReader라는 이름으로 수정했습니다.